### PR TITLE
ci(travis): utilize stages and add Android and iOS builds

### PR DESCRIPTION
**Pull request by hypery2k**
* Added native android build
* Added iOS build in Travis
* Utilize stages

### CODE REVIEW

**Comment by hypery2k** [comment]
@sis0k0 Any chance to get it merged?

**Comment by sis0k0** [comment]
Great job, @hypery2k! I have only one concern, but I may be wrong. I think the building of the tests app is missing. It's removed from the config (https://github.com/NativeScript/nativescript-angular/pull/1176/files#diff-354f30a63fb0907d4ad57269548329e3L47) and I don't see it in the updated version. Is that intentional?

**Comment by hypery2k** [comment]
no it wasn't, just by accident

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,95 @@
...
+        - QEMU_AUDIO_DRV=none emulator -avd $EMULATOR_NAME -no-window &
+        - nvm install $NODE_VERSION
+        - cd nativescript-angular
+        - npm install
+        - npm run ngc
+        - cd ..
+        - android-wait-for-emulator
+        - adb shell input keyevent 82 &
+      script: 
+        - tns build android
```
Comment (sis0k0) [inline .travis.yml]: I think this should be the last step (after navigating to tests and running npm install)

Comment (hypery2k) [reply .travis.yml]: @sis0k0  the adb shell or the build?

Comment (sis0k0) [reply .travis.yml]: The build

Comment (sis0k0) [reply .travis.yml]: ping @hypery2k  ^

**Comment by sis0k0** [comment]
ping @hypery2k https://github.com/NativeScript/nativescript-angular/pull/1176/commits/c8d028c0f5bf9424c624c5f574db487bb21e5fc1#r167857339

**Comment by hypery2k** [comment]
@sis0k0 Could you trigger your jenkins ci build, also?

File: tests/package.json [hunk 2]
```diff
@@ -1,6 +1,12 @@
 {
   "nativescript": {
-    "id": "org.nativescript.ngtests"
+    "id": "org.nativescript.ngtests",
+    "tns-ios": {
```
Comment (sis0k0) [inline tests/package.json]: Can you remove the platforms here?

Comment (hypery2k) [reply tests/package.json]: no, because than tns build will fail with `version of undefined` errors

Comment (sis0k0) [reply tests/package.json]: Ok, change the versions to 'next' then :)

Comment (hypery2k) [reply tests/package.json]: done ;)

**Comment by sis0k0** [comment]
test

**Comment by hypery2k** [comment]
thanks ;)

**Pull request merged**